Sources/EmbraceConfigInternal/EmbraceConfigurable/RemoteConfig/RemoteConfigFetcher.swift

```diff
@@ -77,7 +77,7 @@ class RemoteConfigFetcher {
             URLQueryItem(name: "appId", value: options.appId),
             URLQueryItem(name: "osVersion", value: options.osVersion),
             URLQueryItem(name: "appVersion", value: options.appVersion),
-            URLQueryItem(name: "deviceId", value: options.deviceId),
+            URLQueryItem(name: "deviceId", value: options.deviceId.hex),
```

Comment:
I think we can even remove this. The remote config service is not really using this. Also, we already did that in 5.x
```suggestion
```